edx-platform/utec/lms/templates/footer.html

```diff
@@ -5,11 +5,21 @@
   from django.utils.translation import ugettext as _
   from branding.api import get_footer
   from openedx.core.djangoapps.lang_pref.api import footer_language_selector_is_enabled
+  from openedx.core.djangoapps.user_api.preferences.api import get_user_preference
+  from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 %>
 <% footer = get_footer(is_secure=is_secure) %>
 <% icp_license_info = getattr(settings, 'ICP_LICENSE_INFO', {})%>
 <%namespace name='static' file='static_content.html'/>
 
+<%
+options_modal = configuration_helpers.get_value("MODAL_TOS", {})
+modal_force_display = options_modal.get("modal_force_display", False)
+name_preference_tos = options_modal.get("modal_tos_name_preference", "")
+modal_tos_text = options_modal.get('modal_tos_text', "We have updated our terms of service")
+modal_tos_btn_label = options_modal.get('modal_tos_btn_label', "")
```

Comment:
here you could have a default value as in `modal_tos_text`

Reply:
I decided to remove it because they have a cookie consent